backend/server.js

```
@@ -1,17 +1,113 @@
-require('dotenv').config();
+require("dotenv").config();
 
-const app = require('./app');
-const { connectMongo } = require('./config/db2');
+const app = require("./app");
+const { connectMongo } = require("./config/db2");
 
-require('./config/clients');
+require("./config/clients");
 
+const app = express();
```

Biome (2.3.14)

[error] Shouldn't redeclare 'app'. Consider to delete it or rename it.

'app' is defined here:


(lint/suspicious/noRedeclare)